Documentation/tutorial/howto_customize_docfx_flavored_markdown.md

```diff
@@ -0,0 +1,146 @@
+How-to: Customize DocFX Flavored Markdown
+=========================================
+
+Customize Renderer
+------------------
+
+DocFX Flavored Markdown introduced @Microsoft.DocAsCode.Dfm.IDfmCustomizedRendererPartProvider from v2.17.
```

Comment:
read again, original "introduced" is correct

---
In reply to: [118661879](https://github.com/dotnet/docfx/pull/1705#discussion_r118661879) [](ancestors = 118661879)